### DISCUSSION TITLE
Uninstall NominatimNLpBackend (mobile app, installed via F-Droid)

### DISCUSSION THREAD

**Discussion by Osmandfan** [osm-search/Nominatim#2956] [Q&A]
I installed nominatim on my fairphone 3 with e/ OS to get a GPS problem fixed. It did not work out well.
How can I uninstall Nominatim from my phone. I can't find it in apps. Tried revo uninstaller for mobiles but that does not find it either.   
Thank you for your time and help.

**Comment by mtmail** [comment]
How did you install it? You might be the first person ever who installed such a server software on a phone. Which data did you import? How did you use it, or what was the aim with "GPS problem"? I'm not even sure how Nominatim is related to GPS in this case.

**Reply by Osmandfan** [reply]
I downloaded it form Fdroid It says NominatimNLpBackend. Someonen suggested by installing this the GPS problem was fixed on his phone so I tried it out.

**Reply by mtmail** [reply]
I see.
https://f-droid.org/en/packages/org.microg.nlp.backend.nominatim/ lists points to https://github.com/microg/NominatimGeocoderBackend
Use that issue tracker instead. It's unrelated to the project here.

**Reply by Osmandfan** [reply]
Thank you for the advise. I am not very technical. Do you perhaps have a link were I can post a question Can't seem to find it on the second link you gave me.

**Reply by mtmail** [reply]
https://github.com/microg/NominatimGeocoderBackend/issues
